### ISSUE TITLE
Issue with the `user_role` table

### ISSUE THREAD

**Issue by algorythm** [hold12/CDIO_del1#16]
When trying to add roles to a user in the program, I get this SQL error:

```
[DBConnection::query]: Cannot add or update a child row: a foreign key constraint fails 
(`CDIO_del1`.`user_role`, CONSTRAINT `user_role_ibfk_1` FOREIGN KEY (`user_id`) 
REFERENCES `user` (`id`) ON DELETE CASCADE)com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: 
Cannot add or update a child row: a foreign key constraint fails 
(`CDIO_del1`.`user_role`, CONSTRAINT `user_role_ibfk_1` FOREIGN KEY (`user_id`) 
REFERENCES `user` (`id`) ON DELETE CASCADE)
```

(in one line, has been split to multiple lines for better readability)

**Comment by freyahelstrup** [comment]
Hvor i koden sker fejlen? Jeg forestiller mig den sker hvis du prøver at indsætte en userid i user_role,  som ikke findes i user.

**Comment by algorythm** [comment]
Fixed in 3b43b6298e69c27cd0a7e17661383578e6542944.
